basic/build.sbt

```diff
@@ -0,0 +1,24 @@
+val scalaMeterVersion = "0.7-SNAPSHOT"
+
+lazy val basic = Project(
```

Comment:
Looking good!
Could you also add a short comment here, explaining what this project definition is, i.e.:
- it is the simplest possible use of ScalaMeter
- it allows running ScalaMeter tests as part of the test suite
- when the `test` command is run, ScalaMeter benchmarks are run along the tests from other test frameworks, such as ScalaCheck